introduction/bloco_8/dia_1/script.js

```diff
@@ -0,0 +1,35 @@
+// Exercise 1
+const functionFirstOrder = (fullName) => {
+  return {
+    fullName,
+    email: fullName.toLowerCase().split(' ').join('_')+`@trybe.com`,
```

Comment:
Que tal ao invés de usar `split()` e `join()` você usar `replace()`?
E usar template literals?  :)
```suggestion
    email: `${fullName.toLowerCase().replace(/ /gi, '_')}@trybe.com`,
```